src/hashdump.c

```diff
@@ -56,6 +56,8 @@ static void usage(const char *name)
 	print("\n");
 	print("The filename is the name of a file on the filesystem.\n");
 	print("\n");
+	/* TODO(castulo): add --debug and --quiet to the menu options once we are ready
```

Comment:
This was already addressed on other PR, right?

Reply:
yep, I forgot to remove it